### DISCUSSION TITLE
tiktok wrong counter

### DISCUSSION THREAD

**Discussion by Jan204** [3dproger/AxelChat#565] [General]
Hey,
the counters of tiktok are wrong. someone is over 1000.
![Screenshot 2024-10-08 145803](https://github.com/user-attachments/assets/60bd49ac-60c0-4371-95a4-bb0de7375b42)

**Comment by 3dproger** [comment]
Hey,
thanks for letting me know, I'll try to fix it

**Reply by 3dproger** [reply]
Hello!

I think I managed to fix it. You can download it here: https://github.com/3dproger/AxelChat/releases/tag/0.38.5-beta
